src/pages/blog/blogArchives/blogArchives.scss

```diff
@@ -1,8 +1,21 @@
 #blog-archive-container {
-  display: flex;
-  flex-wrap: wrap;
   padding: 5em 3em;
   margin: 0 auto;
+
+  #blog-archive-header{
```

Comment:
Looks like the below code was copied from the work you were doing on the Blog Archives ticket? Doesn't really apply here yet because there is not `#blog-archive-header` in `blogArchives.js` yet. This styling will probably need to be revisited after that PR is merged, which means this whole PR will probably need to wait.